src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/AddressDtoValidator.java

```diff
@@ -33,6 +33,20 @@ public Errors validate(String parentAddressField, AddressDto addressDto, List<St
         return errors;
     }
 
+    public Errors validateOtherAddressIsNotSupplied(String parentAddressField, AddressDto addressDto, Errors errors, String loggingContext) {
+        StringValidators.checkIsEmpty(addressDto.getPropertyNameNumber(), getQualifiedFieldName(parentAddressField, AddressDto.PROPERTY_NAME_NUMBER_FIELD), errors, loggingContext);
```

Comment:
Been chatting to @markpit , we think this block of checks should be wrapped in 'not null' check, i.e. only do them if the 'addressDto' is non null. If it is null, there is no error to report as that means the alternative address has not been supplied.